input/pagecontent/comparison.md

```diff
@@ -202,6 +202,44 @@ AU Core compliant resources are compliant with IPA requirements for Missing Data
 ##### Suppressed Data
 IPA does not include requirements for Suppressed Data.
 
+##### Additional Must Support elements
```

Comment:
These are just additional requirements. Turn the section into Additional requirements.

Reply:
Done

Reply:
Not done. Instead of the Additional requirements section these were added to the incompatible requirements table. They are not incompatible, they are additional.

Need to have an Additional requirements section for the additional must support requirements.